app/controllers/casa_orgs_controller.rb

```diff
@@ -1,4 +1,6 @@
 class CasaOrgsController < ApplicationController
+  # NOTE: I don't know what auth levels to use here, but am pretty certain it needs SOMETHING.
+  before_action :authenticate_user!
```

Comment:
This will probably eventually need to be `authenticate_all_casa_admin!` but we're not using it for now so this is fine (I'm not sure whether Devise `authenticate_all_casa_admin!` exists/works yet)